install-ast/action.yml

```
@@ -15,12 +15,12 @@ runs:
     - name: Set up dotnet
       uses: actions/setup-dotnet@2216f56ae1eec353f06abd464e2ec435fa5f5d43
       with:
-        dotnet-version: "3.1.x"
+        dotnet-version: "6.0.x"
 
     - name: Install AST
       shell: pwsh
       env:
-        AST_PINNED_COMMIT: "ce87e84a58dff318f62ffe5177bf3e179d815108"
+        AST_PINNED_COMMIT: "9f30f7a267eff7f3fe494c49e5a536a337c8dfde"
```

```suggestion
        AST_PINNED_COMMIT: "9f30f7a267eff7f3fe494c49e5a536a337c8dfde" # v4.0.1
```
Whether this works with Renovate or not, it would be nice for the user.